packages/client-init/typescript/bin/rocketride-init.js

```
@@ -0,0 +1,190 @@
+const PNPM = 'pnpm';
+
+/**
+ * Spawn a command portably (see the PNPM note above).
+ *
+ * @param {string} cmd - The executable.
+ * @param {string[]} args - Its arguments.
+ * @param {object} opts - spawnSync options.
+ * @returns {object} The spawnSync result.
+ */
+function spawnPortable(cmd, args, opts) {
+	if (process.platform === 'win32') {
```

ESLint

[error] 'process' is not defined.

(no-undef)